primefaces-integration-tests/src/test/java/org/primefaces/integrationtests/datatable/DataTable034Test.java

```diff
@@ -43,6 +43,9 @@ public void test(Page page) throws InterruptedException {
 
         Assertions.assertEquals("Language 1", it.getValue());
 
+        it.clear();
```

Comment:
i call clear explizit because this already triggers the first ajax-refresh. after this we need to call `createFragment` again.